pkg/util/sets/string.go

```diff
@@ -44,8 +44,14 @@ func (s Set) InsertAll(items ...string) Set {
 	return s
 }
 
+// Delete removes an item from the set.
+func (s Set) Delete(item string) Set {
+	delete(s, item)
+	return s
+}
+
 // Delete removes items from the set.
-func (s Set) Delete(items ...string) Set {
+func (s Set) DeleteAll(items ...string) Set {
```

Comment:
why change this

Reply:
Most operations delete a single key, it, so it's better to provide a specialized version. It I
is also consistent with the other methods Insert/InsertAll.

Reply:
Prefer no couple unrelated refactor